signing/src/main/java/tech/pegasys/web3signer/signing/bulkloading/AWSBulkLoadingArtifactSignerProvider.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.web3signer.signing.bulkloading;
```

Comment:
Maybe not within the scope of this PR, but I think this class should be in the same package as `BlsKeystoreBulkLoader`.

Reply:
I originally wanted it to place there but then decided to create a subpackage. I've refactored it to `tech.pegasys.web3signer.signing` as per suggestion.